universal-login-commons/src/core/models/Network.ts

```diff
@@ -0,0 +1,34 @@
+import {asEnum} from '@restless/sanitizers';
+
+export type Network
+= '1' | 'mainnet'
+| '3' | 'ropsten'
+| '4' | 'rinkeby'
+| '42' | 'kovan'
+| '8545' | 'ganache';
+
+export const Network = {
+  toNumericId(network: Network): number {
+    switch (network) {
+      case '1':
+      case 'mainnet':
+        return 1;
+      case '3':
+      case 'ropsten':
+        return 3;
+      case '4':
+      case 'rinkeby':
+        return 4;
+      case '42':
+      case 'kovan':
+        return 42;
+      case '8545':
+      case 'ganache':
+        return 8545;
+      default:
+        throw new TypeError(`INvalid network: ${network}`);
+    }
+  },
+};
+
+export const asNetwork = asEnum<Network>(['1', 'mainnet', '3', 'ropsten', '4', 'rinkeby', '42', 'kovan', '8545', 'ganache'], 'Network');
```

Comment:
Move to /core/utils/sanitizers ?

Reply:
I think having model specific sanitizers right with the model is better for code readability. Usually the sanitizer follows the model definition 1-to-1 and it doesn't make sense to split them